src/amo/components/AddonReviewCard/styles.scss

```diff
@@ -65,6 +65,18 @@
 }
 
 .AddonReviewCard-delete {
+  text-align: center;
+  width: 100%;
+
+  .ConfirmButton-buttons {
+    display: block;
+  }
+
+  .ConfirmButton-cancel-button,
+  .ConfirmButton-confirm-button {
+    width: 100%;
```

Comment:
The mocks we have only specify what the *Rate your experience* block should look like. I filed https://github.com/mozilla/addons-frontend/issues/6458 to get a mock for how the listing will look. In any case, maybe we should adjust your patch like this?

```diff
diff --git a/src/amo/components/AddonReviewCard/index.js b/src/amo/components/AddonReviewCard/index.js
index b7bc2185c..a871cf99c 100644
--- a/src/amo/components/AddonReviewCard/index.js
+++ b/src/amo/components/AddonReviewCard/index.js
@@ -386,7 +386,7 @@ export class AddonReviewCardBase extends React.Component<InternalProps> {
                 confirmButtonText={this.confirmButtonText()}
                 id={`${confirmButtonClassName}-${review.id}`}
                 onConfirm={this.onClickToDeleteReview}
-                puffyButtons
+                puffyButtons={verticalButtons}
               >
                 {this.deletePrompt()}
               </ConfirmButton>
diff --git a/src/amo/components/AddonReviewCard/styles.scss b/src/amo/components/AddonReviewCard/styles.scss
index 05ac9f1fa..ae8d74685 100644
--- a/src/amo/components/AddonReviewCard/styles.scss
+++ b/src/amo/components/AddonReviewCard/styles.scss
@@ -65,16 +65,9 @@
 }
 
 .AddonReviewCard-delete {
-  text-align: center;
-  width: 100%;
-
-  .ConfirmButton-buttons {
-    display: block;
-  }
-
   .ConfirmButton-cancel-button,
   .ConfirmButton-confirm-button {
-    width: 100%;
+    margin: 0 6px;
   }
 
   .ConfirmButton-cancel-button,
diff --git a/src/amo/components/RatingManager/styles.scss b/src/amo/components/RatingManager/styles.scss
index f6ea6a455..dfa5e2de2 100644
--- a/src/amo/components/RatingManager/styles.scss
+++ b/src/amo/components/RatingManager/styles.scss
@@ -32,8 +32,29 @@
     justify-content: space-between;
   }
 
-  .AddonReviewCard-ratingOnly .AddonReviewCard-allControls {
-    justify-content: center;
+  .AddonReviewCard-ratingOnly {
+    .AddonReviewCard-allControls {
+      justify-content: center;
+    }
+
+    .AddonReviewCard-delete {
+      text-align: center;
+      width: 100%;
+
+      .ConfirmButton-buttons {
+        display: block;
+      }
+
+      .ConfirmButton-cancel-button,
+      .ConfirmButton-confirm-button {
+        margin: 0;
+        width: 100%;
+      }
+
+      .ConfirmButton-cancel-button {
+        margin-top: 12px;
+      }
+    }
   }
 
   .DismissibleTextForm-delete-submit-buttons,

```

This makes wide buttons on *Rate your experience* only.

**Before my patch**

<img width="343" alt="screenshot 2018-09-28 15 17 48" src="https://user-images.githubusercontent.com/55398/46231731-a04d4600-c332-11e8-8908-388c3fbaa355.png">
<img width="622" alt="screenshot 2018-09-28 15 29 40" src="https://user-images.githubusercontent.com/55398/46232046-83654280-c333-11e8-97a8-a319b23bca4e.png">

<img width="623" alt="screenshot 2018-09-28 15 17 21" src="https://user-images.githubusercontent.com/55398/46231734-a2170980-c332-11e8-9a1a-03364b6c07e8.png">

**After my patch**

<img width="343" alt="screenshot 2018-09-28 15 15 02" src="https://user-images.githubusercontent.com/55398/46231741-ac390800-c332-11e8-96f3-27ae320245f5.png">
<img width="622" alt="screenshot 2018-09-28 15 33 01" src="https://user-images.githubusercontent.com/55398/46232225-1605e180-c334-11e8-9fcf-813a0058a839.png">
<img width="621" alt="screenshot 2018-09-28 15 33 51" src="https://user-images.githubusercontent.com/55398/46232226-1605e180-c334-11e8-9a50-5caba1691c9c.png">

Reply:
Thanks @kumar303 I have adopted your patch and it does fix the issues on all the other views.